monero-outreach-docs/translations/de/merchants/retail_resources_de.md

```diff
@@ -0,0 +1,47 @@
+- **Aufkleber in Tür-/Fenstergröße** - 4" x 1-1/3" Größe eines Adressaufklebers. Avery 94206 oder ähnlich.
+[.svg](https://static.monerooutreach.org/img/MoneroMerchants/We-Accept-Monero-Sign-en-4x1.33.svg)
+- **Weboptimierte Vektorgrafik**
+[.svg](https://static.monerooutreach.org/img/MoneroMerchants/We-Accept-Monero-Sign-en-web.svg) | [60px x 16px .png](https://static.monerooutreach.org/img/MoneroMerchants/We-Accept-Monero-Sign-en-60x16.png)
+
+### _Monero QR-Anzeige für Dein Geschäft_
+---
+
+Dieser Aufsteller wird aus einem Blatt Papier gefaltet, um ein Thekendisplay zu erstellen. Kunden können Deinen QR-Code mit ihrer Wallet scannen, um in XMR zu bezahlen. Auf der Rückseite befindet sich ein Platz, an dem Du den Ablauf für die Annahme von XMR als Spickzettel für Dich und Deine Mitarbeiter auflisten kannst.
+
+- **Verkaufsstellen QR-Code** - 4" x 7"
+[.svg](https://static.monerooutreach.org/img/MoneroMerchants/tabletent-4inx12in-v.svg)
+
+Unsere Vorlage kann als Ausgangspunkt verwendet werden, Du hast die Möglichkeit, sie optisch oder inhaltlich anzupassen. Wenn Du mit Software für Grafikdesign nicht vertraut bist, können Dir diese Dateien fremd erscheinen. Sollte das der Fall sein, suche nach Empfehlungen für einen lokalen Grafikdesigner oder veröffentliche eine Anzeige auf [reddit.com/r/moneromarket](https://www.reddit.com/r/moneromarket/) und beauftragen einen Designer aus der Monero-Community.
+
+Du musst den QR-Code für Deine Monero-Adresse generieren, damit der Kunde mit seiner mobilen Wallet diese scannen kann. Verwende z. B. [the-qrcode-generator.com](https://www.the-qrcode-generator.com/) oder einen anderen bevorzugten QR-Code-Generator, um Deine Adresse einzugeben und eine Datei für den QR-Code zu erzeugen. Das svg-Dateiformat ist ideal für einen klaren Druck, vermeide wenn möglich .jpgs oder .pngs.
+
+Auf der Rückseite dieses Thekendisplays hast Du Platz, um den Ablauf für die Annahme von XMR zu beschreiben. Du kannst Notizen für Dich und Deine Mitarbeiter hinterlassen, damit jede Monero-Transaktion reibungslos verläuft.
+
+Diese Vorlage passt auf Papier im Legal-Format 8,5" x 14" (216 x 356 mm), welche Du selbst ausdrucken kannst. Dafür empfiehlt sich ein Karton mit circa 100lb oder 270g/m². Das dafür notwendige Material solltest du in einem lokalen Copyshop erhalten, welcher es bei Bedarf auch kostengünstig für dich bedrucken kann.
+
+### _Monero Beispiele aus dem Einzelhandel_
+
+### _Mehr erfahren_
+
+- Offizielles Monero Logo: [getmonero.org/press-kit](https://www.getmonero.org/press-kit/)
+- Freier QR-Code Generator: [the-qrcode-generator.com](https://www.the-qrcode-generator.com/)
+- [reddit.com/r/Monero/.../my_life_as_a_monero_vendor_at_monerokon...](https://www.reddit.com/r/Monero/comments/c5f02g/my_life_as_a_monero_vendor_at_monerokon_or_ux/)
+Mein Leben als Händler bei MoneroKon: UX-Probleme für stationäre Geschäfte, die XMR akzeptieren.
+
```

Comment:
I think this translation is also okay. Please, refute me if I'm wrong, but I think the translation is perfect.

Reply:
Thank you very much.